Softeq.XToolkit.Common.Tests/Extensions/ListExtensionsTests/ListExtensionsTests.cs

```diff
@@ -46,5 +47,110 @@ public void AddItem_EmptyListFilledFluent_ReturnsListWithAddedItems()
             Assert.Same(list, result);
             Assert.Equal(3, result.Count);
         }
+
+        [Fact]
+        public void AddRange_EmptyList_AddsItemsToList()
```

Comment:
Looks like `AddRange`, `InsertRange` are rudimental and already provided by BCL: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.insertrange?view=netcore-3.1

Reply:
Yes, you are right, I did not pay attention to that. Don't you remember why these methods were added to toolkit in the first place? I'm going to delete them now

Reply:
@wcoder I checked out and those methods are available for List only, our extensions provide them for IList - I guess that was the point. I will fix test data to be IList instead of List